test/integrations/destinations/appcues/router/data.js

```diff
@@ -0,0 +1,238 @@
+const data = [
+    {
+      "name": "appcues",
+      "description": "Test 0",
```

Comment:
Can we spend some time & update correct descriptions ?

Reply:
@krishna2020 I had looked into this. But it is pretty difficult to know what this test case is for through them, as most of them are pretty similar.